src/Standards/Squiz/Sniffs/Commenting/FunctionCommentThrowTagSniff.php

```diff
@@ -70,7 +78,7 @@ protected function processTokenWithinScope(File $phpcsFile, $stackPtr, $currScop
         $currPos          = $stackPtr;
         $foundThrows      = false;
         while ($currPos < $currScopeEnd && $currPos !== false) {
-            if ($phpcsFile->hasCondition($currPos, T_CLOSURE) === false) {
+            if ($this->isLastScope($tokens[$currPos]['conditions'], $currScope) === true) {
```

Comment:
The previous condition was not sufficient because it was checking _any_ closure. It doesn't work correctly in case nested closures, and doesn't also work for anonymous classes.